README.md

```diff
@@ -353,6 +368,8 @@ This needs to be run once to be able to run any other Terraform commands.
 $ STAGE=sandbox yarn run tf:service:init
 ```
 
+> ⚠️ **Warning**: You need to run `yarn run tf:service:init` **every** time you change `STAGE` or other core environmental setup before you can mutate anything with the stack (like `yarn run tf:service:apply`). Failure to do so will result in bad things like incorrect stage variables applied to an old, stale stage in the underlying Terraform local disk cache.
```

Comment:
Just tried the following after having already worked with the sandbox stage:

Bootstrap new stage
```
STAGE=sandbox-andy aws-vault exec root --no-session -- yarn cf:bootstrap:create

--------

$ aws cloudformation create-stack --capabilities CAPABILITY_NAMED_IAM $(yarn -s cf:bootstrap:_tmpl) $(yarn -s cf:bootstrap:_stack) $(yarn -s cf:_params)
{
    "StackId": "arn:aws:cloudformation:us-east-1:xxxx:stack/cf-simple-reference-sandbox-andy-bootstrap/xxxx"
}
```

```

Initialize tensorflow
```
> 
STAGE=sandbox-andy aws-vault exec root --no-session -- yarn tf:service:init

$ yarn run tf:terraform init $(yarn -s tf:service:_backend) $(yarn -s tf:service:_vars)
$ eval $(yarn -s env) && cd terraform && terraform init -backend-config=bucket=cf-simple-reference-sandbox-andy-terraform-state -backend-config=dynamodb_table=cf-simple-reference-sandbox-andy-terraform-locks -backend-config=region=us-east
-1 -var region=us-east-1 -var service_name=simple-reference -var stage=sandbox-andy
Initializing modules...
- module.serverless
- module.serverless_xray
- module.vpc
- module.serverless_vpc
- module.serverless_canary

Initializing the backend...
Backend configuration changed!

Terraform has detected that the configuration specified for the backend
has changed. Terraform will now check for existing state in the backends.


Do you want to migrate all workspaces to "s3"?
  Both the existing "s3" backend and the newly configured "s3" backend
  support workspaces. When migrating between backends, Terraform will copy
  all workspaces (with the same names). THIS WILL OVERWRITE any conflicting
  states in the destination.

  Terraform initialization doesn't currently migrate only select workspaces.
  If you want to migrate a select number of workspaces, you must manually
  pull and push those states.

  If you answer "yes", Terraform will migrate all states. If you answer
  "no", Terraform will abort.

  Enter a value:
```

My understanding is that there should be no overlap between services but this migration prompt requires a yes to proceed.

Reply:
I think this looks like workspaces from your local setup. Let's not use those and I'd recommend nuking all of `.terraform` and starting over.